src/main/java/menion/android/whereyougo/geo/location/LocationState.java

```diff
@@ -13,6 +13,10 @@
  * see <http://www.gnu.org/licenses/>.
  * 
  * Copyright (C) 2012 Menion <whereyougo@asamm.cz>
+ *
+ * Changes:
+ * Date        Who               Detail
+ * 19.01.2020  Kurly1            Add support for partial WakeLock
```

Comment:
We use the git history to find out this infos. 
Please remove.

Reply:
I removed the change log from the files.